tests/integration/simple_seed_test/test_seed_kms_key_name.py

```diff
@@ -0,0 +1,160 @@
+import json
+import os
+
+from google.oauth2 import service_account
+from google.api_core.exceptions import AlreadyExists
+from google.cloud import bigquery
+from google.cloud import kms
+
+from tests.integration.base import DBTIntegrationTest, use_profile
```

Comment:
We've moved away from the tests in `tests/integration` towards `tests/functional`. They are testing the same functionality, but use a new framework. One of the changes is that we moved from `unittest` to `pytest`. Could you migrate these tests to the new framework? At a cursory glance, I think you'd need to make these changes:
- `GcpKmsAdapter`
  - looks like it would move over as-is
- `TestSimpleSeedKmsKeyName`
  - you could probably jetison `schema` and `project_id` in favor of the `update_project_config()` fixture
  - `setUp` would need to be a `pytest` fixture with `autouse=True`
- `TestSimpleSeedKmsKeyNameBq`
  - you'd need to find a new source for `biqguery_profile()`; I'm not that familiar with it, but I'm sure that's done in other tests in `tests/functional`